src/BenchmarksClient/Workers/SignalRWorker.cs

```diff
@@ -404,6 +407,30 @@ private double GetPercentile(int percent, List<double> sortedData)
             return (1.0 - fractionPart) * sortedData[(int)Math.Truncate(i)] + fractionPart * sortedData[(int)Math.Ceiling(i)];
         }
 
+        private async Task DoNTP()
```

Comment:
Are you literally using the NTP protocol? If not, I suggest we just call this "Time Synchronization" or something. Don't want to confuse people.